packages/lesswrong/components/events/modules/EventCards.tsx

```diff
@@ -117,32 +117,14 @@ const EventCards = ({events, loading, numDefaultCards, hideSpecialCards, hideGro
   const { AddToCalendarIcon, PostsItemTooltipWrapper, CloudinaryImage2, VirtualProgramCard } = Components
   
   // while the data is loading, show some placeholder empty cards
-  if (loading && !events?.length) {
+  if (loading && !events.length) {
```

Comment:
I don't really understand the scenario in which `loading && events.length` would be true. Is it like we have loaded some but not all of the events?

If that's the case I'm inclined to still show the loading screen to prevent screen flicker, but probably not a big deal.

Reply:
Yeah - ex. we've loaded the first 12 events and the user clicked "Load More". In that case I think it's better to keep the original 12 cards visible while the next 12 load.